eZ/Publish/API/Repository/Tests/TrashServiceTest.php

```diff
@@ -684,6 +685,41 @@ public function testFindTrashItems()
         $this->assertEquals(4, $searchResult->totalCount);
     }
 
+    /**
+     * Test for the findTrashItems() method.
+     *
+     * @see \eZ\Publish\API\Repository\TrashService::findTrashItems()
+     * @depends eZ\Publish\API\Repository\Tests\TrashServiceTest::testTrash
+     */
+    public function testFindTrashItemsSortedByDateTrashed()
+    {
+        $this->createTrashItem();
+        sleep(1); // make sure trashed date changed
+        $this->trashContactUs();
+
+        $trashService = $this->getRepository()->getTrashService();
```

Comment:
A call to `$this->getRepository()` should always precede any other calls which modify data because w/o arguments it initializes Repository (probably doesn't happen here as there's a flag preventing that, but too risky maintenance-wise).